# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/ClientWebGLContext.h
# Commit: 6149f5b15c5e
# Full Hash: 6149f5b15c5e278ef15ea36fa896fa8c146ebdca
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2021-01-14 08:32:45
# Regressor Bug: 1640607
# File Overlap Count: 1
# Description:
#   Bug 1640607 - Send SurfaceDescriptors for GPU blitting for video-to-webgl. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D101061
# ==============================================================================

diff -r 29fd3a6a7e27 -r 6149f5b15c5e dom/canvas/ClientWebGLContext.h
--- a/dom/canvas/ClientWebGLContext.h	Wed Jan 13 20:14:54 2021 +0000
+++ b/dom/canvas/ClientWebGLContext.h	Wed Jan 13 21:49:40 2021 +0000
@@ -877,6 +877,11 @@
     EnqueueError(0, format, args...);
   }
 
+  template <typename... Args>
+  void EnqueuePerfWarning(const char* const format, const Args&... args) const {
+    EnqueueError(webgl::kErrorPerfWarning, format, args...);
+  }
+
   void EnqueueError_ArgEnum(const char* argName,
                             GLenum val) const;  // Cold code.
 